recipes.json

```diff
@@ -11,6 +11,22 @@
     {
       "link": "https://www.cupcakes.com",
       "title": "cupcakes"
+    },
+    {
+      "link": "https://",
```

Comment:
Why are the links formatted this way? Or was this just from entering empty links?